packages/web/components/trade-clipboard/index.tsx

```diff
@@ -146,7 +164,7 @@ export const TradeClipboard: FunctionComponent<{
         fetchedRemainingPoolsRef.current = true;
         queries.osmosis?.queryGammPools.fetchRemainingPools();
       }
-    }, []);
+    }, [queries.osmosis?.queryGammPools]);
```

Comment:
@jonator What do you think of enabling `react-hooks/exhaustive-deps` eslint rule? If we want to opt out, we should add a eslint-disable comment. Exhaustive deps are important to avoid unexpected bugs coming from `useEffect`.

Reply:
https://github.com/osmosis-labs/osmosis-frontend/issues/1349

I agree, @sehyunc pointed this out and will include it in his pools page redesign PR.